ConfigenDemo/AppEnvironment/AppEnvironment.swift

```diff
@@ -0,0 +1,22 @@
+// auto-generated by configen
+// to add or remove properties, edit the mapping file: '/Users/johnsanderson/Documents/ConfigGenerator/ConfigenDemo/AppEnvironment.map'.
+// README: https://github.com/theappbusiness/ConfigGenerator/blob/master/README.md
+
+import Foundation
+
+class AppEnvironment {
+
+  let entryPointURL: URL = URL(string: "http://example.com/production")!
```

Comment:
looks like there's a bug in the latest code change. these should all be `static`